client/src/paths/FolderContext.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Flex } from "@chakra-ui/react";
+import React from "react";
+import { Outlet, useLocation, useOutletContext } from "react-router";
+import { Link as ReactRouterLink } from "react-router";
+import { SiteContext } from "./SiteHeader";
+import { Text } from "@chakra-ui/react";
+
+/**
+ * This is the header bar for the editor. All tabs/sub-pages in the editor use its context `EditorContext`. It works for both `/documentEditor` and `/compoundEditor`.
+ */
+export function FolderContext() {
+  const context = useOutletContext<SiteContext>();
+  const location = useLocation();
+  const activitiesPath = `/activities/${context.user?.userId ?? ""}`;
+  const isActivitiesActive = location.pathname === activitiesPath;
+  const isTrashActive = location.pathname.startsWith("/trash");
+
+  const sidePanel = (
+    <Flex
+      minWidth={{ base: "100%", md: "8rem", lg: "12rem" }}
+      maxWidth={{ base: "100%", md: "8rem", lg: "12rem" }}
+      flexShrink={0}
+      align="flex-start"
+      borderRight={{ base: "none", md: "solid 2px black" }}
+      p={{ base: "0px", xl: "10px" }}
+      minHeight={{ base: "fit-content", md: "75vh" }}
```

Comment:
Try making this
`minHeight={{ base: "fit-content", md: "100%" }}`
at least if you want to try my next idea of making this panel not part of the scrollable panel.